### PR TITLE
build: Add rustfmt check

### LINKED ISSUE #9: Enforcing rustfmt
We may want to increase maxwidth to 120.

### PR DESCRIPTION
**Pull request by wojciechkepka**
Should fix #9. It was easier to create a new pull request than rebase the old one so there is that.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -45,3 +44,4 @@ jobs:
           rustup toolchain install 1.34.0
           rustup default 1.34.0
       - run: cargo build --all-targets --all
+
```
Comment (lzutao) [inline .github/workflows/main.yml:47]: stray line?

Comment (wojciechkepka) [reply .github/workflows/main.yml:47]: Hmm, to be honest in my local copy there is no empty line at the end. Don't know why it shows up here.

Comment (wojciechkepka) [reply .github/workflows/main.yml]: Fixed it.

**Comment by wojciechkepka** [comment]
I've squashed the commits so I think all is done here :)

File: directories/README.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ The library provides the location of these directories by leveraging the mechani
 This library is written in Rust, and supports Linux, Redox, macOS and Windows.
 Other platforms are also supported; they use the Linux conventions.
 
+## Minimum Rust version policy
```
Comment (lzutao) [inline directories/README.md:29]: Could you rebase against upstream master branch? This diff looks weird.

Comment (wojciechkepka) [reply directories/README.md]: I'm a bit new to all this but I think I did what you asked for :) Could you check now?

**Comment by lzutao** [comment]
Thanks for your PR. Hope to see you next time!

**Comment by lzutao** [comment]
bors retry

**Pull request merged**